src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write your code here
+const populations = document.querySelectorAll('.population');
+
+let totalValue = 0;
+
+for (let i = 0; i < populations.length; i++) {
+  const value = populations[i].textContent.replace(/,/g, '');
+
+  if (!isNaN(Number(value))) {
+    totalValue += Number(value);
+  }
+}
+
+const averageValue = totalValue / populations.length;
```

Comment:
There is a potential division by zero error here. If there are no elements with the class 'population', `populations.length` will be zero, leading to a division by zero when calculating `averageValue`. Consider adding a check to ensure `populations.length` is not zero before performing the division.